index.js

```diff
@@ -0,0 +1,44 @@
+import populate from './modules/populateFunction.js';
+import tasks from './modules/objects.js';
+import loadTasks from './modules/displayfromLS.js';
+
+const newList = document.querySelector('[data-new-list]');
+const newInput = document.querySelector('[data-new-input]');
+const clear = document.querySelector('.clear');
+
+const saveToStorage = (newTask) => {
+  localStorage.setItem('store_now', JSON.stringify(newTask));
+};
+
+let totaltasks = tasks.length;
+const indexFunc = () => {
+  totaltasks += 1;
+  return totaltasks;
+};
```

Comment:
It would be better in my opinion to pass this variable to the function as an argument instead of directly accessing the variable in the global scope, and also the return statement could be simplified into one line like this `return variable = variable + 1;`

Reply:
Thanks a lot for your feedback @ouasamine. I will make those changes.